lewis/scripts/control.py

```diff
@@ -41,16 +41,20 @@ def show_api(remote, object_name):
     methods = []
 
     for member_name in dir(obj):
-        if not member_name[0] in ('_', ':'):
-            member = getattr(obj, member_name)
-
-            if callable(member):
-                methods.append(member_name)
+        if member_name[0] not in ('_', ':') and member_name not in dir(type(obj)):
+            methods.append(member_name)
```

Comment:
The proxies on the client side are quite uniform. Plain attributes are always properties, so they are in `dir(type(obj))`. Methods on the other hands are only put onto the object itself, so it's not in `dir(type(obj))`.

It might be slightly different on the server side, but the client does not know that currently.